src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -22,6 +23,8 @@ public class ProjectContext
 
         public Project ProjectFile => RootProject.Project;
 
+        public LockFile LockFile { get; }
```

Comment:
@troydai @davidfowl 
Which option do you prefer?

I am in favor of getting the resolved file from the ProjectContext and not duplicating the knowledge on how to find it.